### PR TITLE
Adding configuration for APT unattended upgrades

### PR DESCRIPTION
**Pull request by BjoernT**
Closes-Bug: #802

### CODE REVIEW

**Comment by BjoernT** [comment]
recheck

**Comment by hughsaunders** [comment]
recheck

File: rpcd/playbooks/roles/rpc_support/tasks/configure_unattended_upgrades.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+# Copyright 2014, Rackspace US, Inc.
```
Comment (prometheanfire) [inline rpcd/playbooks/roles/rpc_support/tasks/configure_unattended_upgrades.yml]: desperately needs updating, you should update this every time you update a file with it in it to be honest (not that we are good about it).

**Comment by prometheanfire** [comment]
other than the copyright thing, :+1:

**Comment by BjoernT** [comment]
thanks, copyright fixed. Yeah that date is always an issue after new years eve :-)

**Comment by d34dh0r53** [comment]
:+1:

**Pull request merged**